Yosemite/Yosemite/Actions/WooShippingAction.swift

```diff
@@ -90,4 +90,11 @@ public enum WooShippingAction: Action {
     case loadConfig(siteID: Int64,
                     orderID: Int64,
                     completion: (Result<WooShippingConfig, Error>) -> Void)
+
+    /// Updates shipments for given order
+    ///
+    case updateShipment(siteID: Int64,
+                        orderID: Int64,
+                        shipmentToUpdate: WooShippingUpdateShipment,
+                        completion: (Result<WooShippingUpdateShipmentResponse, Error>) -> Void)
```

Comment:
❓ Could you please share why it's necessary to return the response instead of just the list of shipments?

Reply:
Good question, Huong! I returned `WooShippingUpdateShipmentResponse` as it looked neater than sending back a Dictionary. 

I started sending a `[String: [WooShippingShipment]]` Dictionary in https://github.com/woocommerce/woocommerce-ios/pull/15446/commits/27a57c69857703b210edd3eb5bbfd384d1a95123.

Reply:
Thank you Sharma!

From what I understand `[String: [WooShippingShipment]]` means a list of shipments with IDs and the contents. Should the contents be named `WooShippingShipmentItem` instead? And can we create a new typealias for `WooShippingShipments` to use instead of this dictionary?

Reply:
Great suggestion, Huong! Renaming makes complete sense. Done in https://github.com/woocommerce/woocommerce-ios/pull/15446/commits/83ca647bce98b6f9d957e903d8453ce80ea4447c